src/components/grid-aware/HomePageLargeParagraph/HomePageParagraph.module.css

```diff
@@ -0,0 +1,220 @@
+.text {
+  font-size: 50px;
+  font-style: var(--font-family-merriweather);
+  font-weight: 300;
+  line-height: 130%;
+  text-align: center;
+  width: 1060px;
+}
```

Comment:
I think this comment wasn't properly addressed. It looks like the current revision of this class still sets a hardcode font size as well as an invalid usage of the [`font-style`](https://developer.mozilla.org/en-US/docs/Web/CSS/font-style) property. `font-style`, by the way, is meant to select between normal, italic, or oblique (a special type of italics) styles. Here you have put a `font-family` value in the `font-style`.

Instead of all of this, can you simply use the [`font`](https://developer.mozilla.org/en-US/docs/Web/CSS/font) shorthand property, which combines many of the font properties into a single property, and use one of the [variables we have defined](https://github.com/ShelterTechSF/sheltertech.org/blob/23d20a80c384151d4ee92601b37496f41e51085b/src/stylesheets/global.css#L54-L62) which set all these values to match one of the typographies from the [style guide](https://www.figma.com/file/HLVHC58KwkcVFXkKAuY0ej/ShelterTech.org-Design-System?node-id=47%3A75)?